pyproject.toml

```diff
@@ -43,6 +44,10 @@ kaleido = "0.2.1"
 pygithub = "^2.3.0"
 jupyterlab-spellchecker = "^0.8.4"
 
+[tool.poetry.scripts]
+process_repo_entropy = "almanack.cli:process_repo_entropy"
```

Comment:
Consider adding a test for the CLI from the CLI to help measure the expected output.